packages/devtools/client/src/entries/client/routes/doctor/page.tsx

```diff
@@ -0,0 +1,139 @@
+import { FC } from 'react';
+import { useSnapshot } from 'valtio';
+import { Box, Flex, Heading, Link, Text } from '@radix-ui/themes';
+import {
+  HiLink,
+  HiMiniArchiveBox,
+  HiMiniExclamationCircle,
+  HiMiniInboxStack,
+  HiMiniRectangleStack,
+  HiMiniScale,
+} from 'react-icons/hi2';
+import _ from 'lodash';
+import clsx from 'clsx';
+import logo from './rsdoctor-large.png';
+import { $doctor } from './state';
+import styles from './page.module.scss';
+import { IndicateCard } from '@/components/Card';
+
+const Page: FC = () => {
+  const doctor = useSnapshot($doctor);
+  const costs = _(doctor.summary.costs)
+    .sortBy(['startAt', 'name', 'costs'])
+    .sortedUniqBy('name')
+    .cloneDeep();
+
+  for (const cost of costs) {
+    if (cost.name === 'bootstrap->beforeCompile') {
+      cost.name = 'prepare';
+    } else if (cost.name === 'beforeCompile->afterCompile') {
+      cost.name = 'compile';
+    } else if (cost.name === 'afterCompile->done') {
+      cost.name = 'optimize';
+    }
+  }
+
+  const errors = _.groupBy(doctor.errors, 'description');
+
+  return (
+    <Flex direction="column" align="center" p="4">
+      <Flex
+        wrap="wrap"
+        gap="4"
+        mb="8"
+        justify="center"
+        className={styles.container}
+      >
+        <IndicateCard className={styles.primaryCard}>
+          <IndicateCard.Column>
+            <Text>Rsdoctor</Text>
+            <Flex gap="2">
+              <button type="button">v1.2.0</button>
```

Comment:
May this position can show version of webpack or rspack, this value is more important for users.